worker/uniter/operation/errors.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var (
-	ErrNoStateFile            = errors.New("uniter state file does not exist")
+	ErrNoSavedState           = errors.New("saved uniter state does not exist")
```

Comment:
We should probably add some comments here next time we are around this part of the code-base.

Reply:
done